pkg/apis/servicecatalog/v1alpha1/defaults.go

```diff
@@ -18,13 +18,25 @@ package v1alpha1
 
 import (
 	"github.com/satori/go.uuid"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"time"
 )
 
 func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
+func SetDefaults_ServiceBrokerSpec(spec *ServiceBrokerSpec) {
+	if spec.RelistBehavior == "" {
+		spec.RelistBehavior = ServiceBrokerRelistBehaviorDuration
+	}
+
+	if spec.RelistDuration == nil {
```

Comment:
You should only default relist duration if the relist behavior is "Duration".  It should not be defaulted for manual behavior.